app/controller/Groups.js

```diff
@@ -82,6 +82,7 @@ Ext.define('DevCycleMobile.controller.Groups', {
 		this.groupStore.filter('groupCode', code);
```

Comment:
Ah - my previous pull removed the rider store.  The group store is still in use for holding/displaying the groups